srcpkgs/cross-i686-pc-linux-gnu/template

```diff
@@ -26,351 +25,23 @@ checksum="0ab6c55dd86a92ed561972ba15b9b70a8b9f75557f896446c82e8b36e473ee04
  e2c4114e569afbe7edbc29131a43be833850ab9a459d81beb2588016d2bbb8af
  0c68f5655528aed4f99dae71a5b259edc93239fa899e2df79c055275c21749a1"
 
-nocross=yes
-nopie=yes
-nodebug=yes
-lib32disabled=yes
-create_wrksrc=yes
-hostmakedepends="tar gcc-objc gcc-go flex perl python3"
-makedepends="isl15-devel libmpc-devel zlib-devel"
-depends="${pkgname}-libc-${version}_${revision}"
-nostrip_files="libcaf_single.a libgcc.a libgcov.a libgcc_eh.a
- libgnarl_pic.a libgnarl.a libgnat_pic.a libgnat.a libgmem.a"
-
-if [ "$XBPS_TARGET_LIBC" != "glibc" ]; then
-	broken="glibc crosstoolchain only available on glibc"
-fi
+cross_triplet=${_triplet}
+cross_linux_arch=x86
+# explicitly enable for final gcc, as gfortran does not build without on x86
+cross_gcc_configure_args="--enable-libquadmath"
+cross_glibc_cflags="-O2 -Wno-error -march=i686 -mtune=generic"
```

Comment:
This matches with current cflags but i686 build-profiles specified `-mtune=i686`?